spark/src/main/scala/geotrellis/spark/tiling/MapKeyTransform.scala

```diff
@@ -159,21 +153,15 @@ class MapKeyTransform(val extent: Extent, val layoutCols: Int, val layoutRows: I
     val boundsExtent: Extent = boundsToExtent(bounds)
     val rasterExtent = RasterExtent(boundsExtent, bounds.width, bounds.height)
 
-    /*
-     * Use the Rasterizer to construct  a list of tiles which meet
-     * the  query polygon.   That list  of tiles  is stored  as an
-     * array of  tuples which  is then  mapped-over to  produce an
-     * array of KeyBounds.
-     */
-    val tiles: MSet[(Int, Int)] = MSet.empty
+    /* Use the Rasterizer to construct a list of tiles which meet the query polygon. */
+    val tiles: mutable.Set[SpatialKey] = mutable.Set.empty
```

Comment:
Apparently it's good practice to prefix mutable containers with `mutable`, as opposed to aliasing it like I had done.